tinygrad/mlops.py

```diff
@@ -17,6 +17,15 @@ def forward(self, x, dtype):
   def backward(self, grad_output):
     return grad_output.cast(self.input_dtype)
 
+class Bitcast(Function):
+  __slots__ = "input_shape", "input_dtype"
+  def forward(self, x, dtype):
+    self.input_shape = x.shape
+    self.input_dtype = x.dtype
+    return x.bitcast(dtype)
+  def backward(self, grad_output):
+    return grad_output.bitcast(self.input_dtype).reshape(self.input_shape)
```

Comment:
Why does this reshape on the backwards?

Reply:
Earlier I was also accounting for shape changes. Now I don't need that actually since we don't allow bitcasts that change shape. I'll fix that.